packages/paste-website/src/pages/components/status-badge/index.mdx

```diff
@@ -2,7 +2,7 @@ export const meta = {
   title: 'Status Badge - Components',
   package: '@twilio-paste/status',
   description:
-    'A status should be used to communicate to a user the status of a process or connection to an application or service.',
+    'A Status Badge should be used to communicate the status of a process, or the connection to an application or service.',
```

Comment:
```suggestion
    'A Status Badge communicates the status of a process or connectivity to an application or service.',
```

nitpick here because "connection to an application or service" makes it sound like the Status Badge shows the relationship to an application or service.